tasks.py

```diff
@@ -827,6 +828,8 @@ def unittest(  # noqa: PLR0913
         command += " --failfast"
     if buffer:
```

Comment:
Perhaps?
```suggestion
    if buffer and not pdb:
```

Reply:
I toyed with this idea but I feel it's a bit counterintuitive and against the "explicit better than implicit" principle. Just a personal opinion though, I'm OK to implement it this way if you feel strong about it...

Reply:
Whenever I put a breakpoint in a test and forget to use `--no-buffer` it doesn't output anything and just freezes until I do a `docker stop` on the container. Does the same thing happen with `--pdb`?

Reply:
Indeed, you have to add `--no-buffer` too to get to the interesting part. I take it you prefer to auto-add it as it makes more sense?

Reply:
If the behavior when you forget `--no-buffer` is that it silently fails then I would prefer to add it automatically